apps/hash-graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -101,8 +101,7 @@ where
                 let base_uri = BaseUri::new(row.get(base_uri_index))
                     .into_report()
                     .change_context(QueryError)?;
-                let version: i64 = row.get(version_index);
-                let version = OntologyTypeVersion::new(version as u32);
+                let version: OntologyTypeVersion = row.get(version_index);
```

Comment:
What changed to make this safe?

Reply:
Or was it safe originally

Reply:
We are now implementing `FromSql` for `OntologyTypeVersion`, which does this logic (and fails in the case we have a negative value, which should never happen)

Reply:
Cheers